go/examples/cmd/tutorial/main.go

```diff
@@ -45,34 +45,41 @@ func main() {
 
 	getNewAddressResponse, _, err := client.WalletApi.WalletGetNewAddressPost(ctx, getNewAddressRequest)
 	if err != nil {
-		util.HandleErrorAndExit(fmt.Errorf("couldn't generate address: %v", err))
+		util.HandleErrorAndExit(fmt.Errorf("couldn't generate address: %v", util.ErrorResponseString(err)))
 	}
 
 	fmt.Printf("Jane's address details: %v\n", getNewAddressResponse)
 
 	issueAssetRequest := sdk.IssueAssetRequest{
-		WalletLabel: "bank",
-		RecipientAddress: "q1dxlf6vap2566t8w3z8f5j5lxy9n036zfsaytjve7fedsw6w8c9q9ctrwfz6ryyjwkgvj6tjg70f",
-		AssetId:    200,
-		Amount:     1,
-		ClearValue: true,
-		Memo:       "this is for you, Jane",
+		WalletLabel:      "bank",
+		Authorization:    "123",
+		RecipientAddress: getNewAddressResponse.RecipientAddress,
+		AssetId:          200,
+		Amount:           1,
+		ClearValue:       true,
+		Memo:             "this is for you, Jane",
 	}
 
-	_, err = client.WalletApi.WalletIssueAssetPost(ctx, issueAssetRequest)
+	asyncTask, _, err := client.WalletApi.WalletIssueAssetPost(ctx, issueAssetRequest)
 	if err != nil {
-		fmt.Printf("Could not issue asset. Did you configure the bank's private key?\n")
+		util.HandleErrorAndExit(fmt.Errorf("couldn't issue asset: %v", util.ErrorResponseString(err)))
 	}
 
-	time.Sleep(20)
+	issueTaskStatus, err := util.WaitForAsyncTaskDone(ctx, asyncTask.Id, client)
+	if err != nil {
+		util.HandleErrorAndExit(err)
+	}
+	if issueTaskStatus.Result == "failure" {
```

Comment:
Maybe `!= success`?

Also, can we use constants here somehow instead of hand-typed strings?
I guess this would require a change to OpenAPI, but it would probably be for the best.

Reply:
I've started a few days ago to take a look on how to not do magic strings with the _OpenAPI_ framework, however didn't find a way to do so, maybe somehow with [enums](https://swagger.io/docs/specification/data-models/enums/). I got to admit that I haven't got very far with this investigation. So for now it'll be strings.

> Maybe `!= "success"`?

I wanted to show in this case that we could explicitly get `failure` for the sake of the example.

Reply:
So we can use `enum` but seems like the _Go_ generator is missing this thing at the moment:
https://github.com/OpenAPITools/openapi-generator/issues/2360